metadata/reportssql/common_functions.sql

```diff
@@ -90,4 +90,38 @@ END$$
 
 DELIMITER ; 
 
+-- calculateDurationInDays
```

Comment:
@tmvumbi2 This is a function specific for drug duration, so I think it should go in the drug_functions file.